samples/GuestConfiguration/package-samples/README.md

```diff
@@ -0,0 +1,55 @@
+# Azure Policy Guest Configuration samples
+
+These samples are shared to help authors
+to create their own configurations
+and resource modules
+for us in
+[custom Guest Configuration](https://docs.microsoft.com/en-us/azure/governance/policy/how-to/guest-configuration-create) projects.
+
+The community is welcome to contribute changes to these samples,
+or create new samples.
+However, this is not intended to be an Open Source project
+to manage the content used in Azure Policy as built-in content.
+
+## Configurations
+
+[AdministratorsGroupMembers](./configurations/AdministratorsGroupMembers/AdministratorsGroupMembers.mof)<br>
+[AdministratorsGroupMembersToExclude](./configurations/AdministratorsGroupMembersToExclude/AdministratorsGroupMembersToExclude.mof)<br>
+[AdministratorsGroupMembersToInclude](./configurations/AdministratorsGroupMembersToInclude/AdministratorsGroupMembersToInclude.mof)<br>
```

Comment:
Do each one of the configs/modules not need their own Readme describing what they do and how to consume them?

Reply:
I'm not opposed to it but I don't have hours to take it on yet.